tests/testem.js

```diff
@@ -24,23 +24,49 @@ gpii.tests.universal.testem.getTestemOptions = function (that) {
 };
 
 fluid.defaults("gpii.tests.universal.testem", {
-    gradeNames: ["gpii.testem.coverageDataOnly"],
-    testPages:  "tests/web/html/all-tests.html",
-    sourceDirs: [],
+    gradeNames: ["gpii.testem.instrumentation"],
+    testPages:  ["tests/web/html/all-tests.html"],
     coverageDir: "coverage",
-    generateCoverageReport: false,
-    instrumentSource: false,
-    serveDirs:  ["node_modules", "browserify"],
+    reportsDir: "reports",
+    instrumentationOptions: {
+        "excludes": [],
+        "sources": ["./**/*.js"],
+        "nonSources": [
+            "./*.json",
+            "./**/*.json",
+            "./*.html",
+            "./**/*.html",
+            "./scripts/*.js",
+            "./testData/**/*.js",
+            "./tests/**",
+            "./!(node_modules)/**/*.!(js)",
+            "./!(node_modules)/*.!(js)",
+            "./!(node_modules)/tests/**/*.js",
+            "./!(node_modules)/**/tests/**/*.js",
+            "./!(node_modules)/test/**/*.js",
+            "./!(node_modules)/**/test/**/*.js",
+            "./!(node_modules)/**/webTests/**/*.js",
+            "./!(node_modules)/**/public/lib/**/*"
+        ]
+    },
+    sourceDirs: {
+        gpii: "%gpii-universal/gpii"
+    },
+    contentDirs: {
+        browserify:   "%gpii-universal/browserify",
+        node_modules: "%gpii-universal/node_modules",
+        testData:     "%gpii-universal/testData"
+    },
     invokers: {
         "getTestemOptions": {
             funcName: "gpii.tests.universal.testem.getTestemOptions",
             args:     ["{that}"]
         }
     },
     testemOptions: {
-        "routes": {
-            "/gpii": "instrumented/universal/gpii"
-        }
+        tap_quiet_logs: true,
+        disable_watching: true,
+        skip: "PhantomJS,Opera,Safari"
```

Comment:
I'd like to see IE excluded from this too since our only remaining web tests within universal don't test any user-delivered UIs, and the testem bugs wrt. IE are annoying

Reply:
Done.